README.md

```diff
@@ -1,6 +1,12 @@
 # Flux community
 
-Flux community content
+The [Flux project](https://github.com) aspires to be a vendor-neutral home for GitOps in a Cloud Native world.
+
+Started 2016 to automate deployments at Weaveworks, the project has grown dramatically since then. Especially since the planning of Flux v2  and GitOps Toolkit SDK it became clearer that the scope became more ambitious and that the Flux project has become the home for the Flux family of projects, all solving specific GitOps needs.
```

Comment:
```suggestion
Started 2016 to automate deployments at Weaveworks, the project has grown dramatically since then. Especially since the planning of Flux v2 and GitOps Toolkit SDK it became clearer that the scope became more ambitious and that the Flux project has become the home for the Flux family of projects, all solving specific GitOps needs.
```